sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/README.md

```diff
@@ -0,0 +1,126 @@
+# CoreWCF Azure Queue Storage library for .NET
+
+CoreWCF Azure Queue Storage is the service side library that will help existing WCF services to be able to use Azure Queue Storage to communicate with clients as a modern replacement to using MSMQ.
+
+## Getting started
+
+### Install the package
+
+Install the Microsoft.CoreWCF.Azure.StorageQueues library for .NET with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Microsoft.CoreWCF.Azure.StorageQueues
+```
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
+[Storage Account][storage_account_docs] to use this package.
+
+To create a new Storage Account, you can use the [Azure portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```azurecli
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+### Authenticate the service to Azure
+
+In order to receive requests from the Azure Queue Storage service, you'll need to configure CoreWCF with the appropriate endpoint and credentials.  The [Azure Identity library][identity] makes it easy to add Microsoft Entra ID support for authenticating with Azure services.
+
+```C#
```

Comment:
We should be using the [snippets](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) system for all sample code, which ensures that we're building/testing it to prevent it from going stale.

The other extensions packages do this by hooking snippets into a full end-to-end sample rather than tests, as they also require the ASP.NET environment to be available.   
_([example snippet](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/extensions/Microsoft.Extensions.Azure/samples/Startup.cs#L29))_